pages/login.page.js

```diff
@@ -0,0 +1,29 @@
+import BasePage from '../pages/base.page';
+
+class LoginPage extends BasePage{
+
+    get userNameInput(){return $('#username')}
+    get passWordInput(){return $('#password')}
+    get loginButton(){return $('#submit')}
+    get rememberMe(){return $('#remember-me')}
+    
+
+    async loginAs(userName, passWord){
+
+        await this.userNameInput.setValue(userName);
+        await this.passWordInput.setValue(passWord);
+        await this.loginButton.click();
+
+    }
+
+    async selectRememberMe(userName, passWord){
```

Comment:
This method is duplicating much of the logic defined in `loginAs()`.

Consider narrowing the scope of it by only assigning it the responsibility of selecting the remember me option (as its name imply):

```javascript
async selectRememberMe(){
   await this.rememberMe.click();
``` 

And, if you need to login as well as part of a test, just call your current `loginAs()` method after calling the `selectRememberMe()` method. That way you'll login with that option selected, and you won't have to define the logic twice.